server/src/controllers/course.ts

```diff
@@ -33,4 +57,152 @@ export async function addCourse(req: Request, res: Response): Promise<void> {
       error: error,
     });
   }
-}
\ No newline at end of file
+}
+
+export async function getCourse(req: Request, res: Response): Promise<Response> {
+  try {
+    const courseId: number = Number(req.params.courseId);
+    
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const course: any = await courseService.findCourseById(courseId, null);
```

Comment:
Are you using `any` to access other included models without `@ts-ignore`? If so we should discuss which way we want to handle this.

Reply:
Yes. I tried alternative solution for this problem by extending the course model in the services to have translation and instance array. What do you think about this approach? 7d1f7c83268caf8efca11f005b70ba078329b07b

Reply:
Of the three options mentioned here I think that might be the best one. `any` and `@ts-ignore` seem hacky and to an extent defeat the purpose of using TypeScript.